easybuild/easyblocks/o/openfoam.py

```diff
@@ -87,7 +87,8 @@ def extract_step(self):
             try:
                 contents_installdir = os.listdir(self.installdir)
                 # it's one directory but has a wrong name
-                if len(contents_installdir) == 1 and os.path.isdir(os.path.join(self.installdir, contents_installdir[0])):
+                if len(contents_installdir) == 1 and \
+                   os.path.isdir(os.path.join(self.installdir, contents_installdir[0])):
                     source = os.path.join(self.installdir, contents_installdir[0])
```

Comment:
@wpoely86 better move `source` outside of the `if`, I don't like this use of `\`